platform/security/sts/security-sts-ldaplogin/src/main/java/ddf/ldap/ldaplogin/LdapLoginConfig.java

```diff
@@ -167,9 +171,14 @@ public void setStartTls(String startTls) {
     ldapProperties.put(START_TLS, startTls);
   }
 
-  public void setUserNameAttribute(String userNameAttribute) {
-    LOGGER.trace("setUserNameAttribute called: {}", userNameAttribute);
-    ldapProperties.put(USER_NAME_ATTRIBUTE, userNameAttribute);
+  public void setLoginUserAttribute(String loginUserAttribute) {
+    LOGGER.trace("setLoginUserAttribute called: {}", loginUserAttribute);
```

Comment:
❓ Any validation we need to do here, i.e., not `null`, empty string, any other potential ldap query injection? Should add corresponding unit tests if the code needs to change.

Reply:
It can be added if necessary, though I think it may be outside the scope of this change since none of the other fields are checked at all. There are currently no default values to set anything to inside the code either since all of it is being done in the blueprint at the moment

Reply:
Agree that if we want to add validation to the ldap classes that should probably be another ticket.